spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+
```

Comment:
I think we can at least write some basic tests for http responding successfully to the show view. We should write tests before we merge.